components/HeroSection.js

```diff
@@ -12,7 +10,18 @@ function HeroSection({ main = false, image, inNeed = false }) {
         main && 'pb-20 md:pb-20'
       }`}
     >
-      <div className={main && `bg-image hero`}>
+      {main && (
+        <div className="hero-image-container">
+          <Image
+            alt="Background"
+            className="hero-image"
+            layout="fill"
+            objectFit="cover"
+            src={image}
+          />
+        </div>
+      )}
+      <div className={main && 'hero'}>
```

Comment:
This looks well, it's good to have an image rather than a background image which we had.